graphics/fontconfig/Portfile

```diff
@@ -45,6 +46,13 @@ configure.args              --disable-silent-rules HASDOCBOOK=no
 # We put this into a pre-configure block so it can be evaluated _after_ platform selection.
 pre-configure {
     configure.args-append   --with-add-fonts=[join ${add_fonts} ,]
+    if {${os.platform} eq "darwin" && ${os.major} > 15} {
+		puts "Running on macOS Sierra or newer..."
```

Comment:
Please do not use `puts` as that would not end up in the log files. Either use `ui_msg` or in this case better `ui_info`, which will only be shown in verbose mode.